1.8/administration/categories-and-forums.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: page
+title: "Categories and Forums Management"
+categories: [administration]
+---
+
+## About Categories and Forums
+
+This is most basic guideline how to make and edit your first forums and categories at your forum.
+
+Categories and forums are root of your forum. It is suggested that they should be in some order so users and visitors can get good view of all categories and forums of your forum. There will be explained basic princip of how categories and forums work and what they are for.
+
+## Categories
+
+In Category you can't write anything, you can't open threads or post. They are made just for organisation. Inside them there are forums and sub-forums wich are here for threads and posting.
+
+
+### So what are Categories for then?
+
+Well, you can't just make some forum / subforum without it's parent. They cannot be alone. They needs parent Category in wich they are put. Categories are mostly general names of forums under it.
```

Comment:
I think this should be:

"Well, you can't make a forum / subforum without it's parent. They cannot be alone. They needs parent Category in which they are put. Categories are mostly general names of forums under it."